distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -270,6 +270,15 @@ public String toString() {
 
     public static final DynamicPort RMI_REG_PORT = new DynamicPort(5);
 
+    public static final AbstractIntegrationTest.DynamicPort AMQP_PORT =
```

Comment:
Wonder if we should have these ports in a different range. Might be nice to group related ports together so maybe for this we use 101-103.